byway/src/main/java/com/google/sps/servlets/DestinationsServlet.java

```diff
@@ -0,0 +1,49 @@
+package com.google.sps.servlets;
+
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/destinations")
+public class DestinationsServlet extends HttpServlet {
+
+    public static class UserLocations{ 
+        String start;
+        ArrayList<String> destinations;
+
+        public UserLocations(){
+            this.start = "";
+            this.destinations= new ArrayList<String>();
+        }
+    }
+
+UserLocations places = new UserLocations();
```

Comment:
Fix spacing and make into private variables

Reply:
done